src/dune_rules/modules.ml

```diff
@@ -535,11 +536,16 @@ let impl impl ~vlib =
 
 let rec find t name =
   match t with
-  | Singleton m -> Option.some_if (Module.name m = name) m
-  | Unwrapped m -> Module_name.Map.find m name
-  | Stdlib w -> Stdlib.find w name
-  | Wrapped w -> Wrapped.find w name
-  | Impl { impl; vlib } -> (
+  | Singleton m ->
```

Comment:
Was this change introduced by accident? It seems like a formatting change, but ocamlformat should do those automatically for us.